makiflow/base/maki_entities/input_maki_layer.py

```diff
@@ -60,5 +60,15 @@ def get_params_dict(self):
         """
         return self._named_params_dict
 
+    def get_params_regularize(self):
+        """
+        This code imitates MakiLayer API for the compatibility sake.
```

Comment:
It does not imitate anything, it is actually a new API. The thing is that it does not violates the old one and fully compatible with it.
Delete this comment.